File: Conditionals/OldEnough.py
#Eve Richardson Old Enough Python

num = int(input("How old are you?\n"))

if num <4:
    print("You can not go to school yet.")
elif num >= 18:
    print("You can vote!")
elif num >= 16:
    print("You can drive!")
elif num >= 15:
    print("You can get your learners permit!")
else:
    print("You can go to school!")

File: Conditionals/OldEnough.c
// Eve Richardson, Old Enough C
#include <stdio.h>

int age = 6;

int main(void){
    
 if(age < 4){
        printf("You can not go to school yet.\n");
    }else if (age >= 18){
        printf("Ypu can vote!\n");
    }else if (age >= 16){
        printf("You can drive!\n");
    }else if (age >= 15){
        printf("You can get your learners permit!\n");
    }else{
        printf("You can go to school!\n");
    }

    return 0;
}